packages/apps/shopify-app-remix/src/server/authenticate/admin/scope/revoke.ts

```diff
@@ -37,8 +37,10 @@ export function revokeScopesFactory(
       });
     }
 
-    const scopesInfo = await fetchScopeInformation(admin);
-    return mapFetchScopeInformation(scopesInfo);
+    const scopesDetail = await fetchScopeDetail(admin);
+    return {
+      detail: mapFetchScopeDetail(scopesDetail),
```

Comment:
I updated the return value of `revoke` to match the [one designed with the tag](https://github.com/Shopify/ui-api-design/blob/main/apis/Scopes/README.md#revoke-write_orders-and-write_themes)